csrankings-c.csv

```diff
@@ -25,7 +25,7 @@ C. Siva Ram Murthy,IIT Madras,http://www.cse.iitm.ac.in/~murthy,VjJFBjcAAAAJ
 C. V. Jawahar,IIIT Hyderabad,https://www.iiit.ac.in/~jawahar,U9dH-DoAAAAJ
 C. Y. Roger Chen,Syracuse University,https://eng-cs.syr.edu/our-departments/electrical-engineering-and-computer-science/people/?peopleid=2970,NOSCHOLARPAGE
 C.-C. Jay Kuo,University of Southern California,http://mcl.usc.edu,81d60okAAAAJ
-C.-H. Luke Ong,University of Oxford,https://www.cs.ox.ac.uk/luke.ong,NOSCHOLARPAGE
+C.-H. Luke Ong,Nanyang Technological University,https://dr.ntu.edu.sg/cris/rp/rp01699,NOSCHOLARPAGE

```

Comment:
Need to also update entry for `Chih-Hao Luke Ong`.

Reply:
That's the precise aim of the current pull request:
changing *Chih-Hao* Luke Ong entry. 
Previous change used a different shorter name "Luke Ong", and therefore didn't connect correctly to dblp. 

Thanks.

Reply:
All of these correspond to the same author and all need to be included / updated.

C.-H. Luke Ong 
Chih-Hao Luke Ong
Luke Ong

Reply:
Done now: all three instances of Luke Ong refer to NUT now.

Reply:
I also quickly found this scholar page. Please update: gNFel3QAAAAJ

Reply:
Done now (in [csrankings-c.csv]). 

The other instance of Luke Ong is in [csrankings-l.csv] - not changed, since I don't know how to change in this PR two different files, unfortunately. Also, this third instance of Luke Ong is not needed because DBLP seems not to use this name.